src/main/java/uk/ac/ebi/eva/utils/ConnectionHelper.java

```diff
@@ -70,24 +67,17 @@ public static MongoTemplate getMongoTemplate(String database) throws UnknownHost
 
     public static MongoClient getMongoClient(String hosts, String authenticationDB,
                                              String user, char[] password) throws UnknownHostException {
-        if(mongoClientWithAuthentication == null){
-            mongoClientWithAuthentication = new MongoClient(
+    	return new MongoClient(
```

Comment:
This will create more connections but I guess it is necessary to run the test in parallel...